extension/docs/WebComponentsIntegrationGuide.md

```diff
@@ -513,6 +516,26 @@ A CTP payment with `submitAdditionalPaymentDetailsResponse` field with the respo
   }
 }
 ```
+If you received an action object and need to repeat `submitPaymentDetailsRequest`, you must remove the existing `submitAdditionalPaymentDetailsResponse` custom field. This can be done in one request to CTP as follow: 
+```json
+{
+  "version": "PAYMENT_VERSION",
+  "actions": [
+    {
+      "action": "setCustomField",
+      "name": "submitAdditionalPaymentDetailsRequest",
+      "value": "{\"details\":{\"threeds2.challengeResult\":\"eyJ0cmFuc1...\"}}"
+    },
+    {
+      "action": "setCustomField",
+      "name": "submitAdditionalPaymentDetailsResponse",
+      "value": ""
```

Comment:
is it better to describe it without value ? (unset)

Reply:
Ah correct: https://github.com/commercetools/commercetools-adyen-integration/pull/337/commits/c64bd199642694466b3d96f31fe07f3b4fa21c29